src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
     "FhirServer": {
         "Security": {
-            "Enabled": true,
+            "Enabled": false,
```

Comment:
😮 
I'm sometimes wondering if `false` might be the better state for local debugging.

Reply:
Oops!  thank you.